roles/smallfile-bench/templates/workload_job.yml.j2

```diff
@@ -14,6 +14,7 @@ spec:
       containers:
         - name: benchmark-server
           image: "quay.io/cloud-bulldozer/smallfile:master"
```

Comment:
Should this be latest or master? Obviously changing this to a version number in the long run but atm?

Reply:
both master and latest shall point to the same based on how i setup the triggers in quay.io, but either way out of scope of this PR

Reply:
ack sounds good.